smarty/templates/email/notifier_publication_review.tpl

```diff
@@ -1,9 +1,9 @@
-Subject: [LORIS] Your project proposal has been reviewed
+Subject: [LORIS] A project proposal has been reviewed
 
 Hello,
 
-Your project proposal "{$Title}" has been reviewed.
+This project proposal "{$Title}" has been reviewed.
```

Comment:
This sentence is grammatically incorrect. It should be "The project proposal {$Title} has been reviewed." not "This"